src/java.base/share/classes/java/nio/channels/AsynchronousFileChannel.java

```diff
@@ -396,7 +396,7 @@ public static AsynchronousFileChannel open(Path file, OpenOption... options)
      * @param   metaData
      *          If {@code true} then this method is required to force changes
      *          to both the file's content and metadata to be written to
-     *          storage; otherwise, it need only force content changes to be
+     *          storage; otherwise, it needs only force content changes to be
```

Comment:
This is a correct though rare use of subjunctive mood:  One need not change it.
However, if be desired that we change the mood to indicative, I suggest s/it need only force/it only needs to force/.

Reply:
(`it need only force` is not really subjunctive mood, but rather usage as a modal auxiliary, see https://english.stackexchange.com/a/297235)